docs/refs.bib

```diff
@@ -127,3 +127,22 @@ @ARTICLE{2024ApJ...964...74S
        adsurl = {https://ui.adsabs.harvard.edu/abs/2024ApJ...964...74S},
       adsnote = {Provided by the SAO/NASA Astrophysics Data System}
 }
+@INPROCEEDINGS{2021SPIE11821E..0UA,
```

Comment:
Please remove. This reference is only referred to in the file camera.py, which you can remove.